HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -46,13 +46,145 @@ def customizeHLTFor49147(process):
                 setattr(prod, 'geometry', g) 
     return process 
 
+def customizeHLTfor48921(process):
+    """ This customizer
+        - renames some of the geometry parameters used to fill the CAGeometry:
+            - minZ -> minInner
+            - maxZ -> maxInner
+            - maxR -> maxDR
+        - for pp and HIN hlt setups.
+    """
+ 
+    ca_producers_pp = ['CAHitNtupletAlpakaPhase1@alpaka','alpaka_serial_sync::CAHitNtupletAlpakaPhase1']
+    ca_producers_hi = ['CAHitNtupletAlpakaHIonPhase1@alpaka','alpaka_serial_sync::CAHitNtupletAlpakaHIonPhase1']
+    ca_producers = ca_producers_pp + ca_producers_hi
+
+    for ca_producer in ca_producers:
+        for prod in producers_by_type(process, ca_producer):
+
+            # cell pt cut
+            if hasattr(prod, 'cellPtCut'):
+                cellPtCut = getattr(prod, 'cellPtCut').value()
+                delattr(prod, 'cellPtCut')
+            else:
+                cellPtCut = 0.85
+
+            isPP = (ca_producer in ca_producers_pp)
+                
+            # geometry
+            if hasattr(prod, 'geometry'):
+                geometry = getattr(prod, "geometry")
+            else:
+                geometry = cms.PSet()
+
+            # pair graph of layer pairs
+            if not hasattr(geometry, "pairGraph"):
+                setattr(geometry, "pairGraph", cms.vuint32( 
+                    0, 1, 0, 4, 0,
+                    7, 1, 2, 1, 4,
+                    1, 7, 4, 5, 7,
+                    8, 2, 3, 2, 4,
+                    2, 7, 5, 6, 8,
+                    9, 0, 2, 1, 3,
+                    0, 5, 0, 8,
+                    4, 6, 7, 9))
+            
+            nPairs = int(len(geometry.pairGraph) / 2)
+
+            # dca cut for connections
+            if not hasattr(geometry, "caDCACuts"):
+                if isPP:
+                    setattr(geometry, "caDCACuts", cms.vdouble([0.0918113099491] + [0.420724617835] * 9))
+                else:
+                    setattr(geometry, "caDCACuts", cms.vdouble(0.05, 0.1, 0.1, 0.1, 0.1,
+                                                                0.1, 0.1, 0.1, 0.1, 0.1))
+
+            nLayers = len(geometry.caDCACuts)
+            
+            # theta cut for connections
+            if not hasattr(geometry, "caThetaCuts"):
+                if isPP:
+                    setattr(geometry, "caThetaCuts", cms.vdouble([0.00123302705499] * 4 + [0.00355691321774] * 6))
+                else:
+                    setattr(geometry, "caThetaCuts", cms.vdouble(0.001, 0.001, 0.001, 0.001, 0.002,
+                                                                 0.002, 0.002, 0.002, 0.002, 0.002))
+
+            if not hasattr(geometry, "startingPairs"):
+                if isPP:
+                    setattr(geometry, "startingPairs", cms.vuint32( [i for i in range(8)] + [i for i in range(13,20)]))
```

Comment:
Following https://github.com/cms-sw/cmssw/pull/49147, I believe this should be corrected to
```cms.vuint32( [i for i in range(8)] + [i for i in range(13,19)])```, ie, ```19``` instead of  ```20```.